examples/jabberwockipedia/jabberwockipedia.ts

```diff
@@ -0,0 +1,5 @@
+import { BasicEditor } from '../../bundles/BasicEditor';
+
+const editor = new BasicEditor(document.getElementById('example'));
+editor.loadConfig({ debug: true });
```

Comment:
This does not exist anymore since ... 3 minutes ? :p You now need to `addPlugin(DevTools)`

Reply:
:trollface: